docs/developer_guide/deployment.md

```diff
@@ -30,21 +30,24 @@ scheduler and admission controller. It is primarily intended for developers.
 
 ## Build docker image
 
-Under project root of the `yunikorn-k8shim`, run the command to build an image using the map for the configuration:
+Under project root of the `yunikorn-k8shim`, run the command to build run the command to build YuniKorn Docker images:
+
 ```
 make image
 ```
 
-This command will build an image. The image will be tagged with a default version, image tag and your build architecture. 
-
-**Note** the default build uses a hardcoded user and tag. You *must* update the `IMAGE_TAG` variable in the `Makefile` to push to an appropriate repository. 
+**Note** that the default build uses a hardcoded registry and tag. If you want to build docker image with specific arch, version or registry, you can refer the following command.
```

Comment:
Grammar: Change "If you want to build docker image with specific arch" to "If you want to build Docker images with a specific arch" and "you can refer the following command." to "you can refer to the following command:"